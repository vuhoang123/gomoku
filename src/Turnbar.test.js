import React from 'react';
import TurnBar from './components/turnbar';
import renderer from 'react-test-renderer';

describe('Turnbar', () => {

  it('matches snapshot', () => {

    const tree = renderer.create(
      <TurnBar turn={'X'} winner={'X'} reset={() => { }} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

});
