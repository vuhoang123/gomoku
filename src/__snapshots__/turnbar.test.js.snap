// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board matches snapshot 1`] = `
Array [
  <div
    className="turnbar"
  >
    <p>
      Turn: X
    </p>
  </div>,
  <div
    className="board"
  >
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
      <div
        className="cell"
        onClick={[Function]}
      />
    </div>
  </div>,
  <div
    className="play-stack"
  >
    <button
      onClick={[Function]}
    >
      Sort Asc
    </button>
    <button
      onClick={[Function]}
    >
      Sort Desc
    </button>
    <button
      onClick={[Function]}
    >
      Sort By Time
    </button>
    <p>
      Step sorted by 
      asc
    </p>
  </div>,
]
`;

exports[`Turnbar matches snapshot 1`] = `
<div
  className="turnbar"
>
  <p>
    Winner: X
  </p>
  <button
    onClick={[Function]}
  >
    Reset
  </button>
</div>
`;
